From cb4a8816bc093d52b22e876cac4c4bb5b88aedb2 Mon Sep 17 00:00:00 2001
From: Chuanxiao Dong <chuanxiao.dong@intel.com>
Date: Wed, 5 Sep 2012 13:50:53 +0800
Subject: [PORT FROM MAIN] vold: use SDCARD_RW for secondary storage

BZ: 55162

Use SDCARD_RW permission to the secondary storage, so that the 3rd
party application can use it

Change-Id: Ie8c9b76e1c42f1782c5f50e87f98910668fc5951
Orig-Change-Id: I2eb5a914f1fe2821f6f37d5f55201fb2b861faca
Category: aosp improvement
Domain: Storage Removable_Storage
Origin: internal
Upstream-Candidate: no, proprietary
Signed-off-by: Chuanxiao Dong <chuanxiao.dong@intel.com>
Reviewed-on: http://android.intel.com:8080/75936
Reviewed-by: lab_cactus <lab_cactus@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Sun, Jianhua <jianhua.sun@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 Volume.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Volume.cpp b/Volume.cpp
index 2da7b1d..8b98227 100644
--- a/Volume.cpp
+++ b/Volume.cpp
@@ -447,7 +447,7 @@ int Volume::mountVol() {
         int gid;
 
         if (Fat::doMount(devicePath, getMountpoint(), false, false, false,
-                AID_MEDIA_RW, AID_MEDIA_RW, 0007, true)) {
+                AID_MEDIA_RW, AID_SDCARD_RW, 0007, true)) {
             SLOGE("%s failed to mount via VFAT (%s)\n", devicePath, strerror(errno));
             continue;
         }
-- 
1.7.4.1

